base_import_odoo/models/import_odoo_database.py

```diff
@@ -184,15 +184,19 @@ def _run_import_model(self, context):
         """Import records of a configured model"""
         model = self.env[context.model_line.model_id.model]
         fields = self._run_import_model_get_fields(context)
+        local_fields = fields.keys()
+        extra_fields = []
+        if context.model_line.extra_fields:
+            extra_fields = safe_eval(context.model_line.extra_fields)
```

Comment:
yay, that was exactly what I disliked about the first version

Reply:
... maybe I merged to enthousiastically. Why `safe_eval` and not `.split(',')` + `filtered`?

Reply:
I mimiced what you did some lines below for the "defaults" dictionary.

Reply:
yes, but the defaults must be a dict. Now I need to write valid python code and not just field1,field2 etc. But leave as is, this is not worth raising noise I think after reconsidering